ocpp/v16/call.py

```diff
@@ -133,7 +134,8 @@ class InstallCertificate:
 class RemoteStartTransaction:
     id_tag: str
     connector_id: Optional[int] = None
-    charging_profile: Optional[Dict] = None
+    # Accept Dict for backwards compatibility
+    charging_profile: Union[ChargingProfile, Dict, None] = None
```

Comment:
This should still be marked as an optional parameter when creating an instance of the dataclass. Otherwise, a user must declare `charging_profile=None`. While the objects created are identical, it shouldn't be required to declare.

`charging_profile: Optional[Union[Dict, ChargingProfile]] = None`